### DISCUSSION TITLE
RTL 语言适配

### DISCUSSION THREAD

**Discussion by xiaosansiji** [Tencent/tdesign#343] [General]
- 如果你有 RTL 语言适配的需求，请在此留言，帮助我们判断支持这个特性的优先级
- 如果你有 RTL 适配的经验，希望参与组件库 RTL 语言适配改造，也欢迎告诉我们

**Comment by imwangfan** [comment]
国际化项目中需要适配阿拉伯语言的页面布局，阿拉伯语是从右向左排列，这个在国际化中很重要
